### PR TITLE
T5158: Refactoring the commad sh interfaces counters

### PR DESCRIPTION
**Pull request by mkorobeinikov**
## Change Summary
refactoring the sh interfaces counters command
## Types of changes
* [ ]  Bug fix (non-breaking change which fixes an issue)
* [x]  New feature (non-breaking change which adds functionality)
* [x]  Code style update (formatting, renaming)
* [ ]  Refactoring (no functional changes)
* [ ]  Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
* [ ]  Other (please describe):

## Related Task(s)
*https://phabricator.vyos.net/T5158

## Component(s) name
        src/op_mode/interfaces.py

## How to test
To check, needs to execute the command.
sh interfaces counters  and check drops and errors

## Checklist:
* [x]  I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
* [x]  I have linked this PR to one or more Phabricator Task(s)
* [ ]  I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
* [ ]  My commit headlines contain a valid Task id
* [ ]  My change requires a change to the documentation
* [ ]  I have updated the documentation accordingly

### CODE REVIEW

File: src/op_mode/interfaces.py [hunk 1]
```diff
@@ -368,16 +372,20 @@ def _format_show_summary(data):
 
 @catch_broken_pipe
 def _format_show_counters(data: list):
-    formatting = '%-12s %10s %10s %10s %10s'
-    print(formatting % ('Interface', 'Rx Packets', 'Rx Bytes', 'Tx Packets', 'Tx Bytes'))
+    formatting = '%-12s %10s %10s %10s %10s %10s %10s %10s %10s'
+    print(formatting % ('Interface', 'Rx Packets', 'Rx Bytes', 'Tx Packets', 'Tx Bytes', 'Rx Dropped', 'Tx Dropped', 'Rx Errors', 'Tx Errors'))
```
Comment (c-po) [inline src/op_mode/interfaces.py]: This is python2 syntax - please use tabulate which is the preferred output method in the new op-mode scripts.

**Review by c-po — CHANGES_REQUESTED** [review]

**Review by dmbaturin — CHANGES_REQUESTED** [review]
It looks all good other than that Python2 syntax that is unwieldy and harder to read compared to tabulate. Once you make that change, I'll be happy to merge it.

If you need any help with the tabulate library, let me know.

**Comment by mkorobeinikov** [comment]
I've changed the syntax from python 2 to python 3 using "tabulate".

File: src/op_mode/interfaces.py [hunk 2]
```diff
@@ -368,19 +372,23 @@ def _format_show_summary(data):
...
-            intf['tx_packets'],
-            intf['tx_bytes']
-        ))
-
-    return 0
+    data_entries = []
+    for entry in data:
+            Interface = entry.get('ifname')
+            Rx_Packets = entry.get('rx_packets')
+            Rx_Packets = entry.get('rx_bytes')
```
Comment (sever-sever) [inline src/op_mode/interfaces.py]: Variable names should be in lowercase
Also `Rx_packets` are defined 2 times

File: src/op_mode/interfaces.py [hunk 3]
```diff
@@ -368,19 +372,23 @@ def _format_show_summary(data):
...
+            Tx_Packets = entry.get('tx_packets')
+            Tx_Bytes = entry.get('tx_bytes')
+            Rx_Dropped = entry.get('rx_dropped')
+            Tx_Dropped = entry.get('tx_dropped')
+            Rx_Errors = entry.get('rx_over_errors')
+            Tx_Errors = entry.get('tx_carrier_errors')
+            data_entries.append([Interface, Rx_Packets, Rx_Packets, Tx_Packets, Tx_Bytes, Rx_Dropped, Tx_Dropped, Rx_Errors, Tx_Errors])
```
Comment (sever-sever) [inline src/op_mode/interfaces.py]: `Rx_Packets` used two times
`Interface, Rx_Packets, Rx_Packets,`

**Review by sever-sever — CHANGES_REQUESTED** [review]

File: src/op_mode/interfaces.py (hunk 3 above)
Comment (mkorobeinikov) [reply src/op_mode/interfaces.py]: Done

**Review by c-po — APPROVED** [review]

**Review by sever-sever — APPROVED** [review]

**Pull request merged**